src/wasm/wasm-debug.cpp

```diff
@@ -502,18 +557,21 @@ static void updateCompileUnits(const BinaryenDWARFInfo& info,
                     attrSpec,
                   llvm::DWARFYAML::FormValue& yamlValue) {
                 if (attrSpec.Attr == llvm::dwarf::DW_AT_low_pc) {
```

Comment:
what about high_pc? it appears that there are 2 forms of that: one of which is an extent, and the other if which is an absolute address like low_pc. Have you run into that?

Reply:
Yeah, I am working on the next PR already, and it does handle the two cases of high pc being either relative or not. Takes a little more work.